packages/cacti-copm-core/src/main/proto/generated/openapi/models/asset_account_v1_pb.proto

```diff
@@ -0,0 +1,24 @@
+/*
+  Hyperledger Cacti Plugin - Common Operator Primitives
+
+  Contains/describes the Hyperledger Cacti Common Operator Primitives plugin.
+
+  The version of the OpenAPI document: 2.0.0-rc.2
+
+  Generated by OpenAPI Generator: https://openapi-generator.tech
+*/
+
+syntax = "proto3";
+
+package org.hyperledger.cacti.plugin.cacti.plugin.copm.core;
+
+import "google/protobuf/any.proto";
+
+
+message AssetAccountV1PB {
+
+  string network = 232872497;
```

Comment:
Sorry I don't have context about this structure, but why is these numbers so high?
Usually proto structures start with `1`, `2` and so on...

Reply:
The pb files are generated from openapi, following cactus convention.  The openapi desc is here: https://jenniferlianne.github.io/cacti/references/openapi/cacti-copm-core_openapi/,

Reply:
@jenniferlianne Can you follow the suggestion in https://github.com/OpenAPITools/openapi-generator/issues/9646 to set field numbers starting from `1`? That will make the proto message files more readable.

(I don't want this to block the merge of this PR though. If it's inconvenient, you can revise this in the next PR.)

Reply:
I have updated the openapi specification as requested, however x-protobuf-index doesn't seem to be picked up for 'ref' data types.  This is potentially an issue in the generator.

Reply:
Can you try using `type: object`?

Reply:
It didn't make a difference.  I have checked in the changes in case the protobuf generator changes in the future.